apps/web/src/modules/dao/components/About/DaoDescription.tsx

```diff
@@ -0,0 +1,97 @@
+import { Box, Button, Flex, Text } from '@zoralabs/zord'
+import HTMLReactParser from 'html-react-parser'
+import React, { useLayoutEffect, useMemo, useRef } from 'react'
+import ReactMarkdown from 'react-markdown'
+import rehypeRaw from 'rehype-raw'
+import rehypeSanitize from 'rehype-sanitize'
+import remarkGfm from 'remark-gfm'
+
+import { daoDescription as plainDesciption } from 'src/styles/About.css'
+import { isPossibleMarkdown } from 'src/utils/helpers'
+
+import { UNEXPANDED_BOX_HEIGHT, daoDescription, fadingEffect } from './mdRender.css'
+
+export const DaoDescription = ({ description }: { description?: string }) => {
+  const [isOverHeight, setIsOverHeight] = React.useState(false)
+  const [isExpanded, setIsExpanded] = React.useState(false)
+
+  const textRef = useRef<HTMLDivElement>(null)
+
+  useLayoutEffect(() => {
+    if (
+      textRef.current &&
+      textRef?.current?.scrollHeight > textRef?.current?.clientHeight
+    ) {
+      setIsOverHeight(true)
+    }
+  }, [description])
+
+  const correctedDescription = React.useMemo(() => {
+    if (typeof description === 'string') {
+      // Text processing on the backend (possibly subgraph) will sometimes replace
+      // \n with \\n, which will break markdown.
+      // This effect is intermittent, so this catches any instance where it happens
+      return description.replace(/\\n/g, '\n').replace(/\\r/g, '\r')
+    }
+  }, [description])
+
+  // This regex check is large. Memoizing it for perf
```

Comment:
We actually don't need this logic in `ProposalDescription` we handle both html and markdown with `ReactMarkdown`

```
{description && (
              <ReactMarkdown
                className={proposalDescription}
                rehypePlugins={[rehypeRaw, rehypeSanitize]}
                remarkPlugins={[remarkGfm]}
              >
                {description}
              </ReactMarkdown>
            )}

```

Reply:
Existing DAO descriptions use a much larger font size. If I used that for MD, the box would be massive. 

If I were to run it through the MD Editor, it would break the original styles that existing DAOs have. My solution was to only update the styling if they were using the existing MD feature. 

That way DAOs that don't use the feature continue to get the style they expect, and DAOs that do continue to get 
the style they expect (non-breaking)

Let me know what you think.